doc/source/ray-air/doc_code/torch_computer_vision.py

```diff
@@ -0,0 +1,263 @@
+def main():
+    test(framework="torch", datasource="tfrecords")
+
+
+def test(*, framework: str, datasource: str):
```

Comment:
thanks for adding this function! let's make sure to test all these combinations in CI